stable/mlrun/templates/api-ingress.yaml

```diff
@@ -28,14 +25,15 @@ spec:
 {{- end }}
   rules:
   {{- range .Values.api.ingress.hosts }}
-    - host: {{ .host | quote }}
+    - host: {{ . | quote }}
       http:
         paths:
-        {{- range .paths }}
-          - path: {{ . }}
+          - pathType: ImplementationSpecific
```

Comment:
please make the `pathType` overridable in the values file and keep the `paths` list to allow defining multiple paths as follows (mind the indentation):
```
      hosts:
        - host: chart-example.local
          paths:
            - pathType: ImplementationSpecific
              path: /
```

Reply:
Hi @alonmr, I made the requested changes, in addition I also upgrade the api-chief ingress.